requirements.txt

```diff
@@ -1,6 +1,6 @@
 Flask==0.10.1
 MarkupSafe==0.23
-nltk==2.0.4
+nltk==2.0.5
```

Comment:
Otherwise I get this error message
```
Collecting nltk==2.0.4 (from -r /tmp/requirements.txt (line 3))
  Downloading nltk-2.0.4.zip (1.1MB)
    Complete output from command python setup.py egg_info:
    Downloading http://pypi.python.org/packages/source/d/distribute/distribute-0.6.21.tar.gz
    Extracting in /tmp/tmpdkKrvN
    Now working in /tmp/tmpdkKrvN/distribute-0.6.21
    Building a Distribute egg in /tmp/pip-build-sqfCM2/nltk
    /tmp/pip-build-sqfCM2/nltk/distribute-0.6.21-py2.7.egg
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-sqfCM2/nltk/setup.py", line 32, in <module>
        del sdist.finders[:]
    AttributeError: 'module' object has no attribute 'finders'

    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-sqfCM2/nltk/
ERROR: Service 'web' failed to build: The command '/bin/sh -c pip install -r /tmp/requirements.txt' returned a non-zero code: 1
```